index.html

```diff
@@ -4,9 +4,51 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz App</title>
+    <link rel="stylesheet" href="/stylesheet/default.css" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Quizily</title>
   </head>
   <body>
-    Work in process...
+    <nav class="nav flex-r">
+      <header class="header">
+        <div class="title"><h2>Quizily</h2></div>
+      </header>
+      <div class="nav-links flex-r">
+        <a href="#"><h3>Home</h3></a>
+        <a href="#"><h3>Login</h3></a>
+      </div>
+    </nav>
+
+    <div class="container-main flex-r">
+      <main class="main-content flex-c">
+        <h1>Quizily</h1>
+        <p>
+          Want to test your knowledge of some popular series, play our quiz!
```

Comment:
Change the style of this text since it is not visible.

Reply:
ok i will do that